public/doc/openapi-3-v3.1.json

```diff
@@ -2448,6 +2861,114 @@
         "pattern": "^\\d+$",
         "readOnly": true
       },
+      "RhcConnectionCollection": {
+        "description": "Collection of Red Hat Connector Connections along with the metadata",
+        "type": "object",
+        "properties": {
+          "meta": {
+            "$ref": "#/components/schemas/CollectionMetadata"
+          },
+          "links": {
+            "$ref": "#/components/schemas/CollectionLinks"
+          },
+          "data": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/RhcConnectionRead"
+            }
+          }
+        }
+      },
+      "RhcConnectionCreate": {
+        "type": "object",
+        "properties": {
+          "rhc_id": {
+            "description": "The UUID of the connection",
+            "example": "123e4567-e89b-12d3-a456-426614174000",
+            "type": "string"
+          },
+          "extra": {
+            "description": "Extra data in JSON format",
+            "example": "{\"hello\": \"world\"}",
+            "type": "string"
+          },
+          "source_id": {
+            "description": "Hehe",
+            "type": "string"
+          }
+        }
+      },
+      "RhcConnectionRead": {
```

Comment:
it looks that we requires name of model like: `RhcConnection` ? as Source, Application,.... is stated in Schemas.

and also we need to add model:

```ruby
# /app/models/rhc_connection.rb
class RhcConnection < ApplicationRecord
end
```

and hopefully it will work 🙏